pytest-selenium/tests/test_login.py

```diff
@@ -48,3 +51,81 @@ def test_login_and_logout(selenium, base_url, book_slug, page_slug, email, passw
     # AND: Reloading does not reset the state back to logged in
     selenium.refresh()
     assert user_nav.user_is_not_logged_in
+
+
+@markers.test_case("C477329")
+@markers.parametrize("page_slug", ["preface"])
+@markers.nondestructive
+def test_logout_in_osweb_logsout_rex(selenium, base_url, book_slug, page_slug, email, password):
+    # GIVEN: Rex page is open
+    rex = Content(selenium, base_url, book_slug=book_slug, page_slug=page_slug).open()
+    rex_nav = rex.navbar
+
+    # WHEN: Login Rex with email & password
+    rex_nav.click_login()
+
+    accounts = Login(selenium)
+    accounts.login(email, password)
+
+    # AND: Open osweb url in a new tab
+    rex.open_new_tab()
+    rex.switch_to_window(1)
+
+    sleep(1)
```

Comment:
The `sleep` should be in the object, not the test (unless you're executing specific things like accessibility keyboard navigation).